sessions/test_cookies.py

```diff
@@ -392,3 +399,102 @@ def test_cookie(self):
         response = self.client_send_req(client, req)
         self.assertEqual(response.status, '302',
                          "Unexpected redirect status code")
+
+class CookieTimeout(CookiesNotEnabled):
+
+    tempesta = {
+        'config' :
+        """
+        server ${server_ip}:8000;
+        server ${server_ip}:8001;
+        server ${server_ip}:8002;
+
+        sticky {
+            cookie enforce max_misses=2 timeout=2;
```

Comment:
Redirection mark test can be found in the other file in the same directory: `test_redir_mark.py`. At least the first test case is already implemented there.